src/style.css

```diff
@@ -0,0 +1,125 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html {
+  height: 100%;
+}
+
+body {
+  display: flex;
+  min-height: 100%;
+  background: #fff;
+  justify-content: center;
+  flex-direction: column;
+  align-items: center;
+  font-weight: 400;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.start-wrapper {
+  text-align: center;
+}
+
+.start-wrapper.hidden {
+  display: none;
+}
+
+.start-wrapper h4 {
+  margin-bottom: 25px;
+  font-size: 20px;
+}
+
+.start-wrapper input {
+  margin-right: 5px;
+  padding: 5px 10px;
+  border-radius: 4px;
+  border: 2px solid #999;
+}
+
+.start-game-button {
+  background-color: green;
+  color: #fff;
+  padding: 7px 10px;
+  border: 0 solid transparent;
+  border-radius: 4px;
+  cursor: pointer;
+}
+
+.start-game-button:hover {
+  background-color: darkgreen;
+}
+
+.board {
+  width: 250px;
+  height: 250px;
+  display: none;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-content: center;
+  justify-items: center;
+  align-items: center;
+  margin: 35px 0;
+}
+
+.board.visible {
+  display: flex;
+}
+
+.board.disabled {
+  pointer-events: none;
```

Comment:
Is this still necessary?

Reply:
after finishing the game, i want board to be visible, but not clickable